concurrency/1.1/_index.md

```diff
@@ -0,0 +1,37 @@
+<!-- Template for the root page of a specification release -->
```

Comment:
Remove this first line to get the table formatting that we see with the other specs